final_prep/mod_zero_hero.rb

```diff
@@ -77,8 +143,8 @@
 
 # Reflection
 # What parts were most difficult about this exerise?
-
+# Nested hashes took some time to figure out. Static vs dynamic class variables was new as well.
 # What parts felt most comfortable to you?
-
+#Syntax is starting to sink in. I don't think I had to look up basic formatting questions here.
 # What skills do you need to continue to practice before starting Mod 1?
-
+#I dont have a good grasp of which topics I don't know until I am confronted by a new topic/issue.
```

Comment:
I think that's totally reasonable right now - this is brand new! And this is why you have instructors with a thoughtful curriculum - they will put challenges in front of you that you need to tackle, and provide feedback on how you're progressing.